packages/functional/actions/cart-actions.ts

```diff
@@ -0,0 +1,26 @@
+import { Page } from '@playwright/test';
+
+export async function addEstimatedShippingCosts(
+  page: Page,
+  country: string,
+  state: string,
+  city: string,
+  zip: string,
+): Promise<void> {
+  await page.getByRole('combobox', { name: 'Country' }).click();
+  await page.getByLabel(country).click();
+
+  await page
+    .getByRole('combobox', { name: 'State' })
+    .filter({ hasNotText: 'Select state' })
+    .click();
+  await page.getByLabel(state).click();
+
+  await page.getByLabel('City').fill(city);
+  await page.getByLabel('Postcode').fill(zip);
```

Comment:
🍹 I would make sure that these are the right label:
```suggestion
  await page.getByLabel('Suburb/city').fill(city);
  await page.getByLabel('Zip/Postcode').fill(zip);
```